src/components/flat-table/flat-table.stories.mdx

```diff
@@ -208,6 +208,35 @@ Column width can be set by passing number as a `width` prop to the column header
   </Story>
 </Preview>
 
+### With tuncated cell content
+When setting column widths it is also possible to set the content to `truncate`: this will prevent the content from wrapping 
+and add ellipsis to any content that overflows the given width. By default the `title` will be set to the children if it 
+is a string. However, if the passed children are not a string it is possible to override it by passing in a differnt 
+`title` sting.
```

Comment:
Little spelling mistake here. `sting` should be `string`.

Reply:
Call `The Police`